src/approval/ApprovalController.ts

```diff
@@ -208,9 +321,50 @@ export class ApprovalController extends BaseController<
    * origins.
    */
   getTotalApprovalCount(): number {
-    return this.state[APPROVAL_COUNT_STORE_KEY];
+    return this.state.pendingApprovalCount;
   }
 
+  // These signatures create a meaningful difference when this method is called.
+  /* eslint-disable @typescript-eslint/unified-signatures */
```

Comment:
It may make it somewhat more burdensome to maintain, but I think it does make the method easier to use. We can always get rid of the signatures in the future, if it's too much. Since the implementation is very permissive with its parameters, it shouldn't be breaking.